### ISSUE TITLE
Exception on Android

### ISSUE THREAD

**Issue by rws08** [Lyokone/flutterlocation#276]
**Describe the bug**
``` shell
Fatal Exception: java.lang.ArrayIndexOutOfBoundsException: length=7; index=9
       at com.lyokone.location.LocationPlugin$3.onNmeaMessage + 303(LocationPlugin.java:303)
       at android.location.LocationManager$GnssStatusListenerTransport$GnssHandler.handleMessage + 1419(LocationManager.java:1419)
       at android.os.Handler.dispatchMessage + 102(Handler.java:102)
       at android.os.Looper.loop + 154(Looper.java:154)
       at android.app.ActivityThread.main + 6776(ActivityThread.java:6776)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run + 1496(ZygoteInit.java:1496)
       at com.android.internal.os.ZygoteInit.main + 1386(ZygoteInit.java:1386)
```

**Tested on:**
Android SM-J530S OS 7.0

``` shell
some logs
```

**Comment by Lyokone** [comment]
Hey ! thanks for the report, not sure why the NMEA message is not complete for this device. I'll add a condition to check that :)

**Comment by Lyokone** [comment]
The fix has been released with version 2.5.0, thanks for the report :)
